components/BugZap/BugZapGame.js

```diff
@@ -132,7 +132,8 @@ class BugZapGame extends React.Component {
     this.fps = 20;
     this.setState({
       characterAnimationIndex: [1,2,3,4,5,6,7],
-    });    // reset characters to default state
+      splashAnimationIndex: [3,4,5],
```

Comment:
Why are aren't you using the character animationIndex function? and animation stuff? ie. character.animationIndex('SPLASH').